docs/assets/style/theme.css

```diff
@@ -2,3 +2,19 @@
 .md-tabs {
   background-color: rgb(89, 73, 167) !important;
 }
+h1 {
+  font-weight: bold;
+  padding-top: 50px;
+}
+h2 {
+  font-weight: bold;
+  padding-top: 50px;
+}
+h3 {
+  font-weight: bold;
+  padding-top: 50px;
+}
+h4 {
+  font-weight: bold;
+  padding-top: 50px;
+}
```

Comment:
Notice that this might only affect the docs at https://dasch-swiss.github.io/dsp-tools/ . If you want it to take effect on docs.dasch.swiss, you may have to change things on the docs repo too. (I'm not sure though.)